tla-io/src/main/scala/at/forsyte/apalache/io/lir/PrettyWriter.scala

```diff
@@ -473,14 +465,19 @@ class PrettyWriter(
         wrapWithParen(parentPrecedence, op.precedence, group(doc))
 
       case OperEx(op @ TlaOper.apply, NameEx(name), args @ _*) =>
+        val (decls, newArgs) = extractDecls(args)
```

Comment:
Do you have an example of what does not work exactly? For instance, this one works in TLC: https://gist.github.com/konnov/e7df8586a9865027dc4abfc07188e060

Reply:
Right. This is because operators are not values in TLA+